packages/csv-parser/src/helper/excel-sheets.ts

```diff
@@ -0,0 +1,94 @@
+class ExcelCsvTransformer {
```

Comment:
Probably, some refactoring should be done to minimize code duplication with the Google Sheets specific code in `spreadsheets.ts` (also, some things should be renamed) - this is just a first draft/proof of concept.